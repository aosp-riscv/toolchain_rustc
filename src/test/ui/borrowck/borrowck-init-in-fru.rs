#[derive(Clone)]
struct Point {
    x: isize,
    y: isize,
}

fn main() {
    let mut origin: Point;
    origin = Point { x: 10, ..origin };
<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
    //~^ ERROR use of possibly uninitialized variable: `origin` [E0381]
=======
    //~^ ERROR use of possibly-uninitialized variable: `origin` [E0381]
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
    origin.clone();
}
