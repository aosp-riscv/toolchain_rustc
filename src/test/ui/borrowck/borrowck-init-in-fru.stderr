<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
error[E0381]: use of possibly uninitialized variable: `origin`
  --> $DIR/borrowck-init-in-fru.rs:9:5
   |
LL |     origin = Point { x: 10, ..origin };
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of possibly uninitialized `origin.y`
=======
error[E0381]: use of possibly-uninitialized variable: `origin`
  --> $DIR/borrowck-init-in-fru.rs:9:5
   |
LL |     origin = Point { x: 10, ..origin };
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of possibly-uninitialized `origin.y`
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0381`.
